middlewares/index.js

```diff
@@ -0,0 +1,23 @@
+function getCookie(cookie) {
+  let result = {};
+  let arr = cookie.split(';');
+  arr.map(el => {
+    const cookieStr = el.split('=');
+    result[cookieStr[0]] = cookieStr[1];
+  });
+  return result;
+}
+
+export const cookieParser = function(req, res, next) {
```

Comment:
Please check keys of result array. They have redundant space inside.

Reply:
added